test/unit/traversing.js

```diff
@@ -508,9 +508,17 @@ test("siblings([String]) - jQuery only", function() {
 });
 
 test("children([String])", function() {
-	expect(2);
-	deepEqual( jQuery("#foo").children().get(), q("sndp", "en", "sap"), "Check for children" );
-	deepEqual( jQuery("#foo").children("#en, #sap").get(), q("en", "sap"), "Check for multiple filters" );
+	expect( 3 );
+
+	deepEqual( jQuery( "#foo" ).children().get(), q( "sndp", "en", "sap" ),
+		"Check for children" );
+	deepEqual( jQuery( "#foo" ).children( "#en, #sap" ).get(), q( "en", "sap" ),
+		"Check for multiple filters" );
+	deepEqual(
+		jQuery( "#foo, #en" ).children().get(),
```

Comment:
"...and we would have gotten away with it too, if it wasn't for that meddling comma operator!"